### DISCUSSION TITLE
German Entity Names instead of English

### DISCUSSION THREAD

**Discussion by Joinsider** [greghesp/ha-bambulab#383] [General]
I don't know if anyone else is bothered by it or I am the only one, but I always get German entity names like _sensor.p1s_..._druckfortschritt_ instead of _sensor.p1s_..._printing_progress_. I don't know if I just misconfigured my integration, but I get those entity names even though my printer and app etc. is set to English only HA is set to german. Is this a feature the integration offers or is this just a bambulabs thing that can't be changed?

**Comment by AdrianGarside** [comment]
Is it just this integration or any integration that has german localization available? The choice of language to display is all handled by core home assistant. So I'd expect you'd see if for multiple integrations.

**Reply by Joinsider** [reply]
Ok thanks for your reply wasn't clear to me
